README.md

```diff
@@ -303,6 +303,20 @@ default value for `-repotoken`.
 
 You can use the `-v` flag to see verbose output.
 
+
+## Gitlab CI
+
+Store your Coveralls API token as an [Environment Variable](https://docs.gitlab.com/ee/ci/variables/#create-a-custom-variable-in-the-ui).
+
+```
+COVERALLS_TOKEN=your_token_goes_here
```

Comment:
Can you remove this? The variable is defined through the GitLab UI, not declared like this.